Kinvey.Core/Realtime/RealtimeRouter.cs

```diff
@@ -152,6 +152,14 @@ internal void SubscribeCollection(string collectionName, KinveyRealtimeDelegate
             string appKey = (KinveyClient.RequestInitializer as KinveyClientRequestInitializer).AppKey;
             string channel = appKey + Constants.CHAR_PERIOD + Constants.STR_REALTIME_COLLECTION_CHANNEL_PREPEND + collectionName;
             AddChannel(channel, callback);
+
+            // In the case of collection subscription, in addition to creating a
+            // channel for the collection, another channel should be created for
+            // this collection and the active user, since certain ACL rules may
+            // not allow collection-wide subscription, but may allow this user
+            // access to the update (see MLIBZ-2223 for more information).
+            string activeUserChannel = channel + Constants.CHAR_PERIOD + Constants.STR_REALTIME_USER_CHANNEL_PREPEND + Client.SharedClient.ActiveUser.Id;
```

Comment:
Yup, that's how we have it in JS too. I'd build those channel names in a separate function, rather than concatenate them inplace,  but that's a different topic :)